docs/core/preview3/tools/dotnet-sln.md

```diff
@@ -0,0 +1,62 @@
+---
+title: dotnet-sln command | Microsoft Docs
+description: The dotnet-sln command provides a convenient option to add, remove, and list projects in a solution file.
+keywords: dotnet-run, CLI, CLI command, .NET Core
+author: spboyer
+ms.author: spboyer
```

Comment:
if you want me to be responsible for this topic, you can put my alias as ms.author (mairaw)